plugins/foreman_ansible/2.x/index.md

```diff
@@ -145,21 +145,13 @@ If this is the first plugin you're installing, please see the [plugin
 repository section]({{site.baseurl}}plugins/#2.2Packageinstallation) to set up the
 repository first.
 
-To install Foreman Ansible, run the following:
+You can use foreman-installer to install Foreman Ansible:
 
-* Red Hat based (RHEL, CentOS...)
+      # foreman-installer --enable-foreman-plugin-ansible --enable-foreman-proxy-plugin-ansible
 
-      # yum install tfm-rubygem-foreman_ansible
+Restarting Foreman might be needed after installing the package. To run Asnible playbooks as Job Templates, you will also need Remote Execution:
```

Comment:
Asnible -> Ansible. The yum package should also restart the foreman application. If not, I think that's a bug.